site/content/docs/commands/pipeline-override.en.md

```diff
@@ -0,0 +1,39 @@
+# copilot pipeline override
+```console
+$ copilot pipeline override
+```
+
+## What does it do?
+Scaffold Infrastructure as Code (IaC) extension files for a pipeline.
+The generated files allow you to extend and override the Copilot-generated AWS CloudFormation template.
+You can edit the files to change existing resource properties, and delete
+or add new resources to an environment's template.
+
+### Learn more
+
+To learn more check out the guides for overriding with [YAML Patches](../developing/overrides/yamlpatch.md) and the
```

Comment:
Shouldn't these three (cdk, taskedef, yamlpatch) also have `en` in them?
`../developing/overrides/yamlpatch.en.md`, etc?